subprojects/gradle/build-checks/src/main/kotlin/com/avito/android/build_checks/internal/unique_app_res/Resource.kt

```diff
@@ -2,7 +2,7 @@ package com.avito.android.build_checks.internal.unique_app_res
 
```

Comment:
In internal package but public in Kotlin.
I just want to show that this class is not intended to direct usage in build scripts.

Reply:
Fixed